src/components/Search/Search.js

```diff
@@ -41,8 +46,12 @@ const Search = ({ query }) => {
   const handleSubmit = (e) => {
     e.preventDefault()
 
-    propertiesURL(searchQuery)
-    searchQuery && searchForProperties(searchQuery)
+    if (patt.test(searchQuery)) {
+      workOrderUrl(searchQuery)
```

Comment:
At the moment we send the user to a work order URL if the query matches our test. But what if they enter an 8 digit number which does not refer to any existing work order e.g. 00000000? I think we'd need something to stop them seeing a 404 in that case. Might be worth checking out what the old app did here.

Reply:
Yeah I agree here. If only React had some `rescue_from` functionality like ruby does. Worth an investigation